docs/layers/geojson-layer.md

```diff
@@ -80,69 +43,111 @@ Whether to draw an outline around polygons (solid fill). Note that
-`getElevation` accessor.
+Extrude Polygon and MultiPolygon features along the z-axis if set to
+true. The height of the drawn features is obtained using the `getElevation` accessor.
 
+- Default: `false`
 
-#### `lightSettings` (Object, optional) **EXPERIMENTAL**
-
-This is an object that contains light settings for extruded polygons.
-Be aware that this prop will likely be changed in a future version of deck.gl.
+#### `wireframe` (Boolean, optional)
 
+- Default: `false`
 
-## Callbacks
+Whether to generate a line wireframe of the hexagon. The outline will have
+"horizontal" lines closing the top and bottom polygons and a vertical line
+(a "strut") for each vertex on the polygon.
 
-#### `onHover` (Function, optional)
+Remarks:
+* These lines are rendered with `GL.LINE` and will thus always be 1 pixel wide.
+* Wireframe and solid extrusions are exclusive, you'll need to create two layers
+  with the same data if you want a combined rendering effect.
+* This is only effective if the `extruded` prop is set to true.
 
-Provides selected feature (and properties) along with mouse event when a
-GeoJson feature is hovered.
+#### `fp64` (Boolean, optional)
 
+- Default: `false`
 
-#### `onClick` (Function, optional)
+Whether the layer should be rendered in high-precision 64-bit mode
 
-Provides selected feature (and properties) along with mouse event when a
-GeoJson feature is clicked.
+#### `lightSettings` (Object, optional) **EXPERIMENTAL**
 
+This is an object that contains light settings for extruded polygons.
+Be aware that this prop will likely be changed in the future versions of
+deck.gl.
 
 ## Accessors
 
 #### `getColor`
 
+- Default: `object => object.strokeColor || [0, 0, 0, 255]`
+
 Called to retrieve the point or line color for a GeoJson feature, depending
 on its type.
 
+#### `getFillColor` (Function, optional)
+
+- Default: `object => object.fillColor || [0, 0, 0, 25]`
+
+Called to retrieve the polygon fill color for a GeoJson feature.
+
+Note: This accessor is only called for `Polygon` and `MultiPolygon` features.
 
 #### `getRadius`
 
-Note: This accessor is only called for `Point` and `MultiPoint` features.
+- Default: `object => object.radius || object => object.size || 1`
+
+Called to retrieve the radius of `Point` and `MultiPoint` feature.
 
+#### `getWidth`
 
-#### `getStrokeWidth`
+- Default: `object => object.strokeWidth || 1`
 
 Called to retrieve the line width for a GeoJson feature.
 
 Note: This accessor is called for `LineString` and `MultiLineString`
 features. It is called for `Polygon` and `MultiPolygon` features if the
 `stroked` attribute is true.
 
-
-#### `getFillColor` (Function, optional)
-
-Called to retrieve the polygon fill color for a GeoJson feature.
-
-Note: This accessor is only called for `Polygon` and `MultiPolygon` features.
-
-
 #### `getElevation` (Function, optional)
 
-- Default: `object => object.elevation`
+- Default: `object => object.elevation || 1000`
```

Comment:
What happens if object.elevation === 0? We probably need to look at this in many places.

An option could be to use object.elevation | 1000 instead of || 1000.